lib/carto/configuration.rb

```
@@ -0,0 +1,71 @@
+module Carto
+end
+
+module Carto::Configuration
```

A general though:

Do you think it would be wise to have a check for environment variables format? For example, if a path variable is set starting with a `/` (a very typical error I think) it might be wise to fail early.